recipe/meta.yaml

```diff
@@ -54,6 +54,7 @@ requirements:
     - pamela
     - typing_extensions
     - adlfs
```

Comment:
i'm not sure whether this should be a run requirement (same as for `s3fs`). these packages are only relevant if you choose to use a blob storage backend of the corresponding cloud provider. for most users these are probably irrelevant and i can't don't anyone needs all of them installed in parallel.

i'd suggest to remove those packages to avoid having dependency conflicts in the future. i also suggest to to include `gcsfs` because of that reason.

Reply:
in fact, quetz already provides a useful error message when the package is not present.

https://github.com/mamba-org/quetz/blob/aab5799b5887471942c4e8a574e30db31aa6fd71/quetz/pkgstores.py#L411-L414

Reply:
I think we should (at least) have a run_constrained then on these packages. At least for google and azure we need a certain minimum version.

Reply:
@wolfv, I moved all the `*fs` packages into the `run_constrained` section. for `psycopg2` i couldn't find a pinning anywhere, so I just removed it for now.